### PR TITLE
[refactor] #3901: delete NewParameter isi and use only SetParameter

### LINKED ISSUE #3901: [suggestion] Move `NewParameter` into Executor
We should consider removing `NewParameter` ISI and only allow defining new parameters in Executor. During Executor migration old parameters will be removed and new ones can be added. The reason is that having an ISI doesn't make sense for an operation that can only be executed by the blockchain admin

### PR DESCRIPTION
**Pull request by Asem-Abdelhady**
## Description
`NewParameter` is only used during genesis generation. it does not make sense to have it as ISI. Now the admin will generate the parameters during configuration and he can give permessions for other users to allow them to put parameters. 

### Linked issue

Closes #3901

### Checklist

- [X] I've read `CONTRIBUTING.md`
- [X] I've used the standard signed-off commit format (or will squash just before merging)
- [ ] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

File: configs/swarm/genesis.json [hunk 1]
```diff
@@ -127,43 +127,43 @@
...
+        "SetParameter": "?WSVAccountMetadataLimits=1048576,4096_ML"
       },
       {
-        "NewParameter": "?WSVDomainMetadataLimits=1048576,4096_ML"
+        "SetParameter": "?WSVDomainMetadataLimits=1048576,4096_ML"
       },
       {
-        "NewParameter": "?WSVIdentLengthLimits=1,128_LL"
+        "SetParameter": "?WSVIdentLengthLimits=1,128_LL"
       },
```
Comment (mversic) [inline configs/swarm/genesis.json:155]: can we drop the `WSV` prefix? what is it's use?

File: configs/swarm/genesis.json [hunk 2]
```diff
@@ -188,5 +188,5 @@
       }
     ]
   ],
-  "executor": "./executor.wasm"
+  "executor": "./configs/swarm/executor.wasm"
```
[the remark this answers is not in the document]
Comment (Asem-Abdelhady) [reply configs/swarm/genesis.json:191]: i didn't generate inline in the last amend let me fix this

**Comment by mversic** [comment]
we need a test that shows how we can create a new parameter in executor

File: configs/swarm/genesis.json (hunk 1 above)
Comment (mversic) [reply configs/swarm/genesis.json:155]: @0x009922

File: configs/swarm/genesis.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (0x009922) [reply configs/swarm/genesis.json]: The executor path is relative to the genesis file, so it should be `./executor.wasm`

File: configs/swarm/genesis.json (hunk 1 above)
Comment (0x009922) [reply configs/swarm/genesis.json:155]: Well, if we are speaking in the context of #4028, then... `SetParameter` should not accept some magically constructed string. It should accept a hard-coded-into-data-model enum of possible on-chain parameters.

As for prefix, I don't think it is needed. In the issue there are comments about naming.

Comment (mversic) [reply configs/swarm/genesis.json:155]: created #4325

**Comment by mversic** [comment]
> we need a test that shows how we can create a new parameter in executor and then update it's value

let's say that we want to define a parameter like `max_accounts_per_domain` in the executor and every time we register an account we also check for this parameter

**Pull request closed without merging**